corporate/tests/test_stripe.py

```diff
@@ -1887,6 +1887,10 @@ def test_update_billing_method_of_current_plan(self) -> None:
         plan.refresh_from_db()
         self.assertEqual(plan.charge_automatically, False)
 
+        realm_audit_log = RealmAuditLog.objects.filter(event_type=RealmAuditLog.REALM_BILLING_METHOD_CHANGED).order_by('-id')[0]
```

Comment:
Can you add a test for the same in 1885 as well? Also no need for the new line in top. Also check the acting_user